vocali-genesis/admin-portal
# DBT-1327: Payment history

## Context
The purpose of this PR is to add a reuseable table with pagination and use this table for displaying the user payment invoices. 





## How to Test
- if you already have a subscription please delete it from the supabase admin dashboard `subscriptions` table.
- Now from the frontend deploy preview of this PR create a new subscription,  then go to `settings/subscriptions`, you should see the subscription invoices table, clicking the receipt button should open the receipt on stripe.

** Note: ** The invoices table will have only 1 invoice on creating a new subscription, to add more invoices here is sql statement that I use to duplicate the invoices, you can run this sql query in the SQL Editor on [Supabase staging.](https://supabase.com/dashboard/project/ppxzhhufvrffpvgraeyh/sql/new)
```sql
INSERT INTO invoices( 
   user_id, subscription_id, amount, metadata, invoice_id, invoice_pdf, invoice_url, currency
)
SELECT user_id, subscription_id, amount, metadata, invoice_id, invoice_pdf, invoice_url, currency FROM invoices limit 1 offset 0;
```







## Proof of Concept
